frontend/packages/core/src/card.tsx

```diff
@@ -126,20 +121,104 @@ const BaseCardContent = styled.div<SpacingProps>`
   ${spacing}
 `;
 
-const StyledCardContent = styled(BaseCardContent)({
+const StyledCardContentContainer = styled.div({
   "> .MuiPaper-root": {
     border: "0",
     borderRadius: "0",
   },
+  overflow: "hidden",
+});
+
+const BaseCardActionArea = styled(CardActionArea)<SpacingProps>`
+  ${spacing}
+`;
+
+const StyledCardActionArea = styled(BaseCardActionArea)({
+  ":hover": {
+    backgroundColor: "#F5F6FD",
+  },
+
+  ":active": {
+    backgroundColor: "#D7DAF6",
+  },
 });
 
+const StyledExpandButton = styled(IconButton)({
+  width: "32px",
+  height: "32px",
+  color: "#3548D4",
+});
+
+interface CardContentCollapsibleProps {
+  /** The text to show in the collapse action area when the card content is collapsed/not collapsed.
+   * By default, this is "See Less" for true and "See More" for false.
+   */
+  dualStateTitle: { true: string; false: string };
+  /** The icon to show in the collapse action area when the card content is collapsed/not collapsed.
+   * By default, this is an ArrowUp icon for true and an ArrowDown icon for false.
+   */
+  dualStateIcon: { true: React.ReactElement; false: React.ReactElement };
+}
+
 interface CardContentProps extends SpacingProps {
   children?: React.ReactNode | React.ReactNode[];
+  /** Make the card content collapse at a set max height. The default is false. */
+  collapsible?: boolean;
+  /** The props for the collapse action if collapsible is enabled */
+  collapseAction?: CardContentCollapsibleProps;
+  /** The max height of the card content. The default is none. */
+  maxHeight?: number;
 }
 
-const CardContent = ({ children, ...props }: CardContentProps) => (
-  <StyledCardContent {...props}>{children}</StyledCardContent>
-);
+const CardContent = ({
+  children,
+  collapsible = false,
+  collapseAction = {
+    dualStateTitle: { true: "See Less", false: "See More" },
+    dualStateIcon: { true: <KeyboardArrowUpIcon />, false: <KeyboardArrowDownIcon /> },
+  },
+  maxHeight,
+  ...props
+}: CardContentProps) => {
+  const ref = React.useRef(null);
+  const [showExpand, setShowExpand] = React.useState(false);
+  const [expanded, setExpanded] = React.useState(true);
+
+  const defaultMaxHeight = maxHeight || "none";
+
+  React.useEffect(() => {
+    if (ref.current.scrollHeight > defaultMaxHeight) {
+      setShowExpand(true);
+      setExpanded(false);
+    }
+  }, [defaultMaxHeight]);
+
+  return (
+    <BaseCardContent {...props} ref={ref}>
+      <StyledCardContentContainer style={{ maxHeight: expanded ? "none" : defaultMaxHeight }}>
+        {children}
+      </StyledCardContentContainer>
+      {collapsible && showExpand && (
+        <StyledCardActionArea padding={1} onClick={() => setExpanded(!expanded)}>
+          <Grid container alignItems="center" spacing={1} justify="center">
+            <Grid item>
+              <Typography variant="body4" color="#3548D4">
```

Comment:
This is minor but it's a bit odd that the icon weight is heavier than the text. Also, should we disable the hover over on the icon since the whole row highlights also?

Reply:
> This is minor but it's a bit odd that the icon weight is heavier than the text

I tried changing the font-weight but that didn't do anything. If I make the icon smaller (currently it's 24px) to 20px but I think that then it looks too small in proportion to the text

<img width="500" alt="Screen Shot 2021-10-05 at 12 07 39 PM" src="https://user-images.githubusercontent.com/39421794/136060996-09f56819-1a1e-4b66-ab37-2ca942afe437.png">